src/components/form/fields/identifierField.jsx

```diff
@@ -103,7 +103,7 @@ class IdentifierField extends React.Component {
             <Input
                 id={inputId}
                 name={path}
-                type={value.type}
+                type='text'
```

Comment:
`value.type` was either `tel`, `email` or `text`. This can function as a filter, so now we only use `text`.